addons/xbmc.python/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.26.0" provider-name="Team Kodi">
+<addon id="xbmc.python" version="3.0.0" provider-name="Team Kodi">
   <backwards-compatibility abi="2.1.0"/>
```

Comment:
Is 2.1.0 still correct? Just asking...

Reply:
nope, it isn't and it will be set to 3.0.0 as well at a later date.
but if we would change it now, you'd end up with 0 addons in kodi.

Reply:
Okay, thanks for explaining.

Reply:
If this is set to 3.0.0, one can't have a single add-on version which is compatible with Matrix and Leia.

Reply:
it's been discussed a while ago and the most favored solution to handle py2/py3 compatibility is:
https://forum.kodi.tv/showthread.php?tid=323369&pid=2688320#pid2688320

Reply:
> You do not have permission to access this page.
That topic seems to be in a restricted section...